.ruby-version

```diff
@@ -1 +1 @@
-jruby-9.2.0.0
+jruby-9.2.5.0
```

Comment:
👍 

this covers that bug that our colleagues mentioned in their own jruby debugging